extensions/ql-vscode/src/view/variant-analysis/__tests__/VariantAnalysisHeader.spec.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
```

Comment:
Is the `__` in dir names a convention when using jest? e.g. `__tests__`, `__mocks__` etc.

Reply:
Sort of; `__tests__` is [the default value](https://jestjs.io/docs/configuration#testmatch-arraystring) for `testMatch`, but I can't really find a specific page documenting this convention. The `__mocks__` naming is also from the Jest documentation, specifically about [setting up Jest with Webpack](https://jestjs.io/docs/webpack#handling-static-assets).

I think it makes sense to follow these defaults and I personally prefer the tests to be close to the components. This convention also makes it easy to distinguish between Mocha and Jest tests.